docs/timezone.md

```diff
@@ -127,11 +127,11 @@ Effectively, this is the canonicalized version of whatever `timeZoneIdentifier`
 
 ## Methods
 
-### timeZone.**getOffsetNanosecondsFor**(_instant_: Temporal.Instant) : number
+### timeZone.**getOffsetNanosecondsFor**(_instant_: Temporal.Instant | string) : number
```

Comment:
I'm not convinced for the TimeZone methods. Authors should generally not call those, so there's no improved ergonomics. On the other hand, it's easy to forget this extra step in custom time zones, especially since Temporal itself will never pass non-normalized arguments, leading to surprising bugs when someone calls the methods directly for some weird reason.

Reply:
I agree with this somewhat, since I did not change the Calendar methods for exactly this reason. But I do think that it's _more_ likely that the TimeZone methods will be called by user code than Calendar methods, especially getNextTransition, getPreviousTransition, getOffsetStringFor, getOffsetNanosecondsFor which don't have any corresponding easier-to-use method on Instant or DateTime. So for the sake of consistency, I'd prefer to keep this as-is.

If you don't agree, would you be OK with merging this for the time being and raising your concern on #592 or in a new issue? Alternatively I could remove the commit with the TimeZone methods from this pull request, but I'd rather stick with the current consensus.

Reply:
So I'd leave as-is and consistent.

Reply:
#1030